packages/storage-plus/src/map.rs

```diff
@@ -118,6 +118,8 @@ mod test {
 
     const ALLOWANCE: Map<(&[u8], &[u8]), u64> = Map::new("allow");
 
+    const TRIPLE: Map<(&[u8], &[u8], &[u8]), u64> = Map::new("triple");
```

Comment:
It would be good to test with non `&[u8]` items, as in the issue.
But that will likely fail to compile/pass due to the issue mentioned above. Let's give a harder test case and fix the issues

Reply:
OK.